src/components/renderer/renderer.js

```diff
@@ -44,6 +46,8 @@ export default class Editor extends React.Component {
         <div className='chart'>
           <div ref='chart'>
           </div>
+          <div id="vis-tooltip" className="vg-tooltip">
```

Comment:
Found it: https://github.com/vega/vega-tooltip/blob/master/docs/creating_your_tooltip.md#step-3-create-a-tooltip.